### ISSUE TITLE
Question: how does omnivore deal with duplicate articles?

### ISSUE THREAD

**Issue by thiswillbeyourgithub** [omnivore-app/omnivore#3786]
Hi,

I sometimes stumble upon a wikipedia page several times in a given year, and want to highlight different part of it. Each time, thinking I'm finished with it I archive the article.

Questions:
1. what happens if I try to save the same articles when the first time it was in the archive?
2. what happens if I try to save the same URL but the content has since changed?
3. Same questions for urls of PDF

In short, I think it would be helpful to have an in depth explanation on how exactly you decided to handle this :)

Thanks!
